Tests/SwiftBSONTests/BSONCorpusTests.swift

```diff
@@ -199,9 +192,10 @@ final class BSONCorpusTests: BSONTestCase {
 
                         let docFromDB = try BSONDocument(fromBSON: dBData)
 
-                        // SKIPPING: native_to_bson( bson_to_native(dB) ) = cB
-                        // We only validate the BSON bytes, we do not clean them up, so can't do this assertion
```

Comment:
I don't think this comment is quite accurate as we can still test we have the right behavior (as I do here) without cleaning up the bytes on document initialization.

I guess we could have written an assertion like
```swift
expect(BSONDocument(fromBSON: dBData).toData()).to(equal(cB))
```
but I'm not actually sure if modifying the degenerate bytes is expected/desired behavior. it seems like it could potentially mess up round trip-ability with the server. what do you think?

Reply:
I think what we have seems to be the best of both worlds: we only yield correct values but we also preserve round trip-ability, so your assertion LGTM.